commit 4e280d4ca2a550ffff495313836ee23fa2c60f77
Author: Marc Poulhiès <dkm@kataplop.net>
Date:   Mon Feb 3 08:14:17 2025 +0100

    Add GMP 6.3.0
    
    GMP 6.2.1 may fail some autoconf test when using GCC master (to-be-15) branch.
    
    Signed-off-by: Marc Poulhiès <dkm@kataplop.net>

diff --git a/packages/gmp/6.3.0/chksum b/packages/gmp/6.3.0/chksum
new file mode 100644
index 00000000..e52daadb
--- /dev/null
+++ b/packages/gmp/6.3.0/chksum
@@ -0,0 +1,8 @@
+md5 gmp-6.3.0.tar.lz db3f4050677df3ff2bd23422c0d3caa1
+sha1 gmp-6.3.0.tar.lz d22966db73fe44928d3abe2a2c1c114c0779353c
+sha256 gmp-6.3.0.tar.lz be5c908a7a836c3a9bd9d62aa58563c5e9e7fef94c43a7f42dbc35bb6d02733c
+sha512 gmp-6.3.0.tar.lz b70f3603ab40270fa6e9a7ee1dc0f91477e45803d3ee2f22af924af89dcd18b483c900685310c9e69cee91dc1b6dec00cfec6a8a111022d0122b5427209a4a2d
+md5 gmp-6.3.0.tar.xz 956dc04e864001a9c22429f761f2c283
+sha1 gmp-6.3.0.tar.xz b4043dd2964ab1a858109da85c44de224384f352
+sha256 gmp-6.3.0.tar.xz a3c2b80201b89e68616f4ad30bc66aee4927c3ce50e33929ca819d5c43538898
+sha512 gmp-6.3.0.tar.xz e85a0dab5195889948a3462189f0e0598d331d3457612e2d3350799dba2e244316d256f8161df5219538eb003e4b5343f989aaa00f96321559063ed8c8f29fd2
diff --git a/packages/gmp/6.3.0/version.desc b/packages/gmp/6.3.0/version.desc
new file mode 100644
index 00000000..e69de29b
